model/src/w3sic4md.F90

```diff
@@ -388,8 +401,13 @@ SUBROUTINE W3SIC4 (A, DEPTH, CG, IX, IY, S, D)
       IF (INFLAGS2(-2)) ICECOEF6 = MUDD(IX,IY) ! a.k.a. MDN
       IF (INFLAGS2(-1)) ICECOEF7 = MUDT(IX,IY) ! a.k.a. MTH
       IF (INFLAGS2(0 )) ICECOEF8 = MUDV(IX,IY) ! a.k.a. MVS
-
+#ifdef CESMCOUPLED
+      ! It does not look like IC4PARS(1) is initialized anywhere - so
+      ! hard-wiring it here for now
+      IC4METHOD = 8
```

Comment:
@lettie-roach - can you address this?

Reply:
Hi @JessicaMeixner-NOAA -  I put this in because I couldn't figure out how to set this variable properly. I think it should be a namelist variable - it's not specific to CESMCOUPLED

Reply:
Have you tried setting in ww3_grid.inp: 
  &SIC4 IC4METHOD = 8 / 
?

Reply:
@JessicaMeixner-NOAA The line in question for hardcoding the ic4method has been removed.